lib/src/main/java/com/thenewboston/common/http/bankapi/validatordto/ValidatorDTO.kt

```diff
@@ -0,0 +1,27 @@
+package com.thenewboston.common.http.bankapi.validatordto
+
+import com.google.gson.annotations.SerializedName
+
+data class ValidatorDTO(
+    @SerializedName("account_number")
+    val accountNumber: String,
+    @SerializedName("ip_address")
+    val ipAddress: String,
+    @SerializedName("node_identifier")
+    val nodeIdentifier: String,
+    val port: Int,
+    val protocol: String,
+    val version: String,
+    @SerializedName("default_transaction_fee")
+    val defaultTransactionFee: Double,
+    @SerializedName("root_account_file")
+    val rootAccountFile: String,
+    @SerializedName("root_account_file_hash")
+    val rootAccountFileHash: String,
+    @SerializedName("seed_block_identifier")
+    val seedBlockIdentifier: String,
+    @SerializedName("daily_confirmation_rate")
+    val dailyConfirmationRate: Double? = null,
```

Comment:
is dailyconfirmationRate really nullable?

Reply:
[https://www.thenewboston.com/bank-api/validators](url)
as per response example provided